<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="main_layout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>761</height>
     </rect>
    </property>
    <widget class="QPushButton" name="replenish_button">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>пополнить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="withdraw_button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>снять</string>
     </property>
    </widget>
    <widget class="QLabel" name="pict4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="table_result">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="slots">
      <item>
       <widget class="QLabel" name="slot1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slot2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slot3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="spin_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>360</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Spin</string>
     </property>
    </widget>
    <widget class="QLabel" name="balance_label">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>700</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
    <zorder>pict4</zorder>
    <zorder>replenish_button</zorder>
    <zorder>withdraw_button</zorder>
    <zorder>table_result</zorder>
    <zorder>horizontalLayoutWidget</zorder>
    <zorder>spin_button</zorder>
    <zorder>balance_label</zorder>
    <zorder>exit_button</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
